src/logging.h

```diff
@@ -91,6 +94,13 @@ namespace BCLog {
          */
         std::atomic_bool m_started_new_line{true};
 
+        //! Category-specific log level. Overrides `m_log_level`.
+        std::unordered_map<LogFlags, Level> m_category_log_levels GUARDED_BY(m_cs);
```

Comment:
non-show-stopper nit: the `m_category_log_levels` and `m_log_level` are introduced in commit
`logging: add -loglevel configuration option`
and later made thread safe in another two commits:
`logging: make m_log_level data member in Logger class thread-safe`
and
`logging: make m_category_log_levels data member in Logger class thread-safe`.

It would make sense to squash the two fixup commits into the first one that introduces the variables.

Reply:
Re-organized into logical steps.